tests/test_functional.py

```diff
@@ -112,9 +112,9 @@ def test_xls_response(pcic_data_portal, authorized_session_id):
     for i, val in enumerate(['Precip_Climatology', 127.128, 91.2249, 77.3313, 46.2816, 39.6803, 33.1902, 22.3557, 22.448, 38.5025, 72.3281, 144.159, 121.008]):
         assert obs.cell_value(i, 0) == val
     # Check a few of the global attributes
-    assert attributes.cell_value(4, 2) == '-123.283333' #longitude
-    assert attributes.cell_value(1, 2) == '1010066' # station_id
-    assert attributes.cell_value(7, 2) == 'ACTIVE PASS' # station name
+    assert attributes.cell_value(5, 2) == '-123.283333' #longitude
```

Comment:
The addition of "elevation" to NC_GLOBAL, made [here](https://github.com/pacificclimate/pydap.handlers.pcic/commit/f1204cb5e05a988617fbcdb02095212ef88ebf38). The elevation value shows up at the top of the file and pushes everything else down a row.